packages/manager/apps/vrack-services/src/pages/index.tsx

```diff
@@ -86,7 +119,32 @@ export default function ListingPage() {
         />
         {t('createVrackServicesButtonLabel')}
       </OsdsButton>
-      <Datagrid data={data.data} />
+
+      {vrackServicesDeliveringOrders?.length > 0 && (
+        <OsdsMessage className="my-5" type={ODS_MESSAGE_TYPE.success}>
+          {'Vrack Services en cours de livraison'}
```

Comment:
Do we need translation key for this ?

Reply:
https://github.com/ovh/manager/pull/10330/files#diff-ac58046a8aa9f6d4236000a3dd2a5d92f669379249895fdf6ca01d6ce3835fb1

fixed here